MOBILE_API.xml

```diff
@@ -3223,7 +3223,8 @@
         <element name="OnUpdateFileID" value="32789" hexvalue="8015" since="7.0" />
         <element name="OnUpdateSubMenuID" value="32790" hexvalue="8016" since="7.0" />
         <element name="OnAppCapabilityUpdatedID" value="32791" hexvalue="8017" since="7.1"/>
-
+	<element name="OnSeekMediaClockTimerID" value="32792" hexvalue="8018" since="7.1" />
```

Comment:
Please align with previous rows, and also at least change to "since 7.2", because we prepare feature to release >7.1
Also please add all this changes to README.md with corresponding python script